plugins/module_utils/scaleway.py

```diff
@@ -80,6 +90,44 @@ def filter_sensitive_attributes(container, attributes):
     return container
 
 
+class SecretVariables(object):
+    @staticmethod
+    def ensure_scaleway_secret_package(module):
+        if not HAS_SCALEWAY_SECRET_PACKAGE:
+            module.fail_json(
+                msg=missing_required_lib("passlib[argon2]", url='https://passlib.readthedocs.io/en/stable/'),
+                exception=SCALEWAY_SECRET_IMP_ERR
+            )
+
+    @staticmethod
+    def dict_to_list(source_dict):
+        return [
+            dict(key=var[0], value=var[1])
+            for var in source_dict.items()
+        ]
+
+    @staticmethod
+    def list_to_dict(source_list, hashed=False):
+        key_value = 'hashed_value' if hashed else 'value'
+        return {
+            var['key']: var[key_value]
+            for var in source_list
+        }
```

Comment:
Python 2.6 does not support set comprehensions. You can use `set([...])` instead of `{...}` (maybe `set(...)` also works).

Reply:
Isn't Python 2.6 deprecated ? Also I didn't see any unitest using this version (2.7 is the oldest)

Reply:
Should be fixed with : 50c022717622a8f981a87154efbb1b740e8c7c42

Reply:
Yes, since we still support ansible-core 2.11 and that still supports Python 2.6, everything still needs some Python 2.6 compatibility.